scripts/convert_lit_checkpoint.py

```diff
@@ -100,6 +100,12 @@ def copy_weights_gpt_neox(
         state_dict[to_name] = param
 
 
+def maybe_unwrap_statedict(lit_weights: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
```

Comment:
```suggestion
def maybe_unwrap_state_dict(lit_weights: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
```

Reply:
implemented in [169acf6](https://github.com/Lightning-AI/lit-gpt/pull/341/commits/169acf6aed0f50e4f6c929274a7902c46bb4815f)